### ISSUE TITLE
配置文件中的输入宽度大于512时，运行就报错

### ISSUE THREAD

**Issue by junjin8433** [XiaoMi/mace#422]
### System information
官方Docker下转换的模型，安卓下调用

### Model deploy file (*.yml)
```yaml
# The name of library
library_name: ocr_recognize
target_abis: [arm64-v8a]
#target_socs: [sdm845]
model_graph_format: file 
model_data_format: file 
models:
  ocr_recognize: # model tag, which will be used in model loading and must be specific.
    platform: tensorflow
    # path to your tensorflow model's pb file. Support local path, http:// and https://
    model_file_path: /home/Documents/my_model/ocr_recognize/ocr_recognize.pb
    # sha256_checksum of your model's pb file.
    # use this command to get the sha256_checksum: sha256sum path/to/your/pb/file
    model_sha256_checksum: b0813900ac1011d76a65f90752a611a700503f1e9f6854c664fa885e81d0696f 
    subgraphs:
      - input_tensors:
          - input_node/images:0
        input_shapes:
          - 1,512,1024,3
        output_tensors:
          - mobilenetv2/output/cls:0
          - mobilenetv2/output/num:0
        output_shapes:
          - 1,64,128,38
          - 1,64,128,2
    # cpu, gpu or cpu+gpu
    runtime: cpu+gpu
    winograd: 0

```

### Describe the problem
一旦转换时候yaml文件中设置的输入宽度（高度没事）大于512，无论实际输入的尺寸多大，都报如下错误：
![image](https://user-images.githubusercontent.com/15881885/55697490-2e963900-59f4-11e9-9548-685ccdbad109.png)
A/MACE: operator.h:140 Check failed: idx < inputs_.size()

### 附上PB文件
[ocr_recognize.pb.zip](https://github.com/XiaoMi/mace/files/3052554/ocr_recognize.pb.zip)

**Comment by lu229** [comment]
@junjin8433 我这边拿附件中的model文件转换测试了一下，转换还没有走到您说的那一步就出错了，原因是因为model中的这个op:
mobilenetv2/concat_1
input:mobilenetv2/concat_1/values_0:0
 input:mobilenetv2/Slice_1:0
output:mobilenetv2/concat_1:0
其中的input:mobilenetv2/concat_1/values_0:0，在graph中没有其他OP生产这个tensor，请问这个OP的上一个OP是什么呢？

**Comment by junjin8433** [comment]
@lu229 
您好，我在固化的时候调过了upsample op，如下
![image](https://user-images.githubusercontent.com/15881885/55712095-b5610b00-5a20-11e9-82ce-0c527984c124.png)
这个upsample其实就是tf.image.resize_images，这个导致问题的op应该是这个函数里面的，在inference的时候并没有用到，如果您那边还是产生错误，可以把这个op删掉。我这边转换文件的时候并没有报任何错误，我使用的就是官方文档里面的docker环境。
![image](https://user-images.githubusercontent.com/15881885/55712133-c3af2700-5a20-11e9-9788-505d4a7b6ffd.png)

**Comment by lu229** [comment]
@junjin8433 我这边用docker跑了一下，还是会出convert的错误，删掉那个OP之后，我这边将runtime从cpu+gpu改成了cpu，可以正常运行，使用的的命令行： python tools/converter.py run --config=./test_models/ocr_recognize/ocr_recognize.yml  --target_socs=random --model_graph_format=file --model_data_format=file
您那边是最新代码么？还是某个版本的代码？把运行的命令行贴一下？

**Comment by junjin8433** [comment]
@lu229 我用的是0.1的release版本，在更新到最新的版本后，的确出现了和您一样的问题，仔细检查之后发现是tf.nn.softmax()的问题（0.1release版本是正常的）。
![image](https://user-images.githubusercontent.com/15881885/55855263-9df95d80-5b99-11e9-9df3-5542f8f4b592.png)
在删除了出问题的op之后转换可以完成，CPU下还是报相同的错误，GPU下报了如下的错误，应该是opencl的问题
![image](https://user-images.githubusercontent.com/15881885/55855380-eb75ca80-5b99-11e9-9a1b-f94da40f628a.png)

**Comment by lu229** [comment]
@junjin8433 您把删除掉OP的模型再上传一下？ 我这边转换调试看看。嗯GPU我这边也有问题，从错误CL_INVALID_IMAGE_SIZE看，是OPENCL不支持这个图像大小（30720, 16），可以参考一下这个[issue](https://github.com/XiaoMi/mace/issues/309#issuecomment-455051042)

**Comment by junjin8433** [comment]
@lu229 PB在此
[ocr_recognize.pb.zip](https://github.com/XiaoMi/mace/files/3063533/ocr_recognize.pb.zip)

**Comment by lu229** [comment]
@junjin8433 我拿您新上传的pb文件测试了一下，这次转换没有问题，我这边mace_run也没有问题，我想是不是因为咱们测试的方法不一样导致的。我这边是运行脚本在手机上去测试的：python tools/converter.py run --config=./test_models/ocr_recognize/ocr_recognize.yml --target_socs=random --model_graph_format=file --model_data_format=file --round=1 --validate。
您那边是怎么测试的？贴一下命令行？

**Comment by junjin8433** [comment]
@lu229 我是mac系统上用的docker进行的转换(mac上的docker无法把USB映射到docker里面)，然后在安卓上直接调用的模型，所以不是命令行运行的:(

**Comment by lu229** [comment]
@junjin8433 , 明白了，我这边通过上面的命令在安卓上运行没有问题，不好复现您那边的问题，您那边安卓端编译使用的mace是最新版本么？

**Comment by junjin8433** [comment]
@lu229 是的，我没有checkout到release版本，直接使用的pull下来的最新版

**Comment by lu229** [comment]
@junjin8433 我这边没法复现，不好定位您的问题，如果您的代码没有保密需求的话，可以附件上来让我看看？
